apps/system/iperf/iperf_net.c

```diff
@@ -238,7 +238,7 @@ int Nread(int fd, char *buf, size_t count, int prot)
 	while (nleft > 0) {
 		r = read(fd, buf, nleft);
 		if (r < 0) {
-			if (errno == EINTR || errno == EAGAIN || errno == EWOULDBLOCK) {
+			if (get_errno() == EINTR || get_errno() == EAGAIN || get_errno() == EWOULDBLOCK) {
```

Comment:
Better to load the get_errno() return value to an auto  variable and check inside if block to avoid multiple system calls incase of user/kernel mode

Reply:
That is good idea. But it requires to modify the whole system including network stack. At this moment, please let me use this simple logic temporarily. Thanks for the comment.